### ISSUE TITLE
eval() in cec/fscil_trainer.py

### ISSUE THREAD

**Issue by InugYoon** [icoz69/CEC-CVPR2021#11]
Hello, I got some short questions. 

In models/cec/fscil_trainer.py, 
At 175 line is model.eval()
I wonder why model.eval() is used instead of train(). 
Was there any reason not to use batchnorm & Dropout at cec trainer, updating encoder & attn?

**Comment by UntitledPro** [comment]
I think the code pretends to train. Truth is the code uses the saved model in the './param' folder for both training and testing.（；´д｀）ゞ

**Comment by icoz69** [comment]
hi,
that's common in the training of the second stage in deep learning. you may turn on it to observe the performance change.
usually, the second stage is for, e.g., finetuning/episode learning, where data is not very sufficient, freezing the BN is found useful
